packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/avatar.tests.js

```diff
@@ -31,34 +31,37 @@ QUnit.module('ChatAvatar', moduleConfig, () => {
             assert.ok(this.instance instanceof ChatAvatar);
         });
 
-        QUnit.test('should be rendered with empty value by default', function(assert) {
-            assert.strictEqual(this.$element.text(), '');
+        QUnit.test('should be rendered with UU value by default', function(assert) {
+            assert.strictEqual(this.$element.text(), 'UU');
         });
     });
 
     QUnit.module('Name option', () => {
         QUnit.test('should be rendered with correct initials according passed name option value', function(assert) {
             this.reinit({ name: 'User name' });
 
-            assert.strictEqual(this.$element.text(), 'U');
+            assert.strictEqual(this.$element.text(), 'UN');
         });
 
         QUnit.test('name option should be updatable at runtime', function(assert) {
             this.instance.option('name', 'New Value');
 
-            assert.strictEqual(this.$element.text(), 'N');
+            assert.strictEqual(this.$element.text(), 'NV');
         });
 
         [
+            { name: 'onewordname', expectedInitials: 'O' },
+            { name: 'Three word name', expectedInitials: 'TW' },
+            { name: ' New Value', expectedInitials: 'NV' },
+            { name: '   New      Value.   ', expectedInitials: 'NV' },
+            { name: '', expectedInitials: '' },
             { name: 888, expectedInitials: '8' },
             { name: undefined, expectedInitials: '' },
             { name: null, expectedInitials: '' },
-            // TODO: consider scenarios
-            // { name: ' New Name', expectedInitials: 'N' },
-            // { name: NaN, expectedInitials: '' },
-            // { name: Infinity, expectedInitials: '' },
-            // { name: -Infinity, expectedInitials: '' },
-            // { name: { firstName: 'name' }, expectedInitials: '' }
+            { name: NaN, expectedInitials: 'N' },
+            { name: Infinity, expectedInitials: 'I' },
+            { name: -Infinity, expectedInitials: '-' },
+            { name: { firstName: 'name' }, expectedInitials: '[O' }
```

Comment:
Is it right, that you and Anton took a decision process object this way?

Reply:
Yes, we decided that we would handle all values except null and undefined and cast them to a string, similar to how it appears in the author-name element. 

I wrote about it here: https://github.com/DevExpress/DevExtreme/pull/28062#discussion_r1763320347 and https://github.com/DevExpress/DevExtreme/pull/28062#discussion_r1763318640

Please see

Reply:
Sure, I saw that. I just wanted to make sure it was a conscious decision.

Reply:
Our logic is simple: we don't expect such values => we don't want to process it in a specific way